### DISCUSSION TITLE
Individual projections don't appear

### DISCUSSION THREAD

**Discussion by guimaricato** [biomodhub/biomod2#230] [Q&A] [answered]
Hello, everyone. I'm having trouble generating individual projections in the ensemble model. When I run "BIOMOD_EnsembleForecasting", the following message appears, but when I check the "individual_projections" folder, it's empty. Thank you in advance.

**Message:**
```
Creating suitable Workdir...

	> Projecting Species_EMmeanByTSS_mergedData_mergedRun_mergedAlgo ...
	> Projecting Species_EMwmeanByTSS_mergedData_mergedRun_mergedAlgo ...
	> Projecting Species_EMcvByTSS_mergedData_mergedRun_mergedAlgo ...
	> Projecting Species_EMcaByTSS_mergedData_mergedRun_mergedAlgo ...
```

**Script:**
```
Proj_Model_EM <- BIOMOD_EnsembleForecasting(bm.em = Model_EM,
                                                  bm.proj = Proj_Model,
                                                  proj.name = "Current_EM",
                                                  models.chosen = "all",
                                                  metric.binary = "all",
                                                  metric.filter = "all",
                                                  compress = TRUE,
                                                  output.format = ".img",
                                                  nb.cpu = 1)
```

**Comment by rpatin** [comment, marked as answer]
Hello @guimaricato,
Thank you for reporting :pray: 
In your folder `Current_EM` you should see a `.img` file, with all your projections stacked. If you want projection written as separate files within folder `individual_projection` you have to use argument `do.stack = FALSE` in `BIOMOD_EnsembleForecasting` (or `BIOMOD_Modeling`)
Best,
Rémi

**Reply by guimaricato** [reply]
Thank you @rpatin, it worked. And thanks for the package too, the new version is amazing.
